src/AddTodoForm.jsx

```diff
@@ -1,8 +1,16 @@
-export default function AddTodoForm() {
+export default function AddTodoForm({ onAddTodo }) {
```

Comment:
The following code would be better:
```
const handleAddTodo = (event) => {
    event.preventDefault();
    const todoTitle = event.target.title.value;
    console.log(todoTitle);
    onAddTodo(todoTitle);
    event.target.title.value = "";
};

```

We converted the function declaration to an arrow function and replaced let with const for the todoTitle variable since its value doesn't change. This makes the code more concise and leverages ES6 syntax.